### PR TITLE
[Product Partnerships]Added azure cognitive sub services

### PR DESCRIPTION
**Pull request by aj-NR**
# Summary

A concise description of the changes being introduced. Please review the pre-merge checklist section to validate this pull request is ready for review and merge. **If it is not ready, please mark the pull request as a draft.**

_The owners of this repo are not experts in the subject matter of the quickstarts. We review for the quickstart to be functional and for security risks. If you are seeking feedback on the content of the quickstart, please seek out a subject matter expert. If you are not an internal NR contributor, we can do our best to connect you with a content reviewer._

## Pre merge checklist

- [X] Did you check you NRQL syntax? - Does it work?
- [X] Did you include a Data source and Documentation reference?
- [X] Are all documentation links publicly accessible?
- [X] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)?
- [X] Did you check your descriptive content for spelling and grammar errors?
- [X] Did you review your content with a subject matter expert? (e.g. a Browser agent quickstart is reviewed with a member of the Browser Agent team)

### Dashboards

- [X] Does the PR contain a screenshot for each of your dashboards?
- [X] Do your screenshots show data?
- [X] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### Alerts

- [X] Did you check that your alerts actually work?

### CODE REVIEW

File: dashboards/azure-anomaly-detector/azure-anomaly-detector.json [hunk 1]
```diff
@@ -0,0 +1,293 @@
+{
+  "name": "Azure Anomaly Detector",
+  "description": null,
```
Comment (caylahamann) [inline dashboards/azure-anomaly-detector/azure-anomaly-detector.json:3]: Could we add a description for the dashboard?

File: dashboards/azure-anomaly-detector/azure-anomaly-detector.json [hunk 2]
```diff
@@ -0,0 +1,293 @@
...
+            "row": 1,
+            "width": 6,
+            "height": 2
+          },
+          "linkedEntityGuids": null,
+          "visualization": {
+            "id": "viz.markdown"
+          },
+          "rawConfiguration": {
+            "text": "# Azure Anomaly Detector\nAnomaly Detector is an AI service with a set of APIs, which enables you to monitor and detect anomalies in your time series data with little machine learning (ML) knowledge, either batch validation or real-time inference."
```
Comment (caylahamann) [inline dashboards/azure-anomaly-detector/azure-anomaly-detector.json:88]: Could this widget describing the dashboard be moved to the top left corner of the dashboard?

File: dashboards/azure-computer-vision/azure-computer-vision.json [hunk 3]
```diff
@@ -0,0 +1,419 @@
...
+            "row": 1,
+            "width": 6,
+            "height": 3
+          },
+          "linkedEntityGuids": null,
+          "visualization": {
+            "id": "viz.markdown"
+          },
+          "rawConfiguration": {
+            "text": "# Azure Computer Vision\nAn AI service that analyzes content in images and video. Boost content discoverability, automate text extraction, analyze video in real time, and create products that more people can use by embedding cloud vision capabilities in your apps with Computer Vision, part of Azure Cognitive Services. Use visual data processing to label content with objects and concepts, extract text, generate image descriptions, moderate content, and understand people's movement in physical spaces."
```
Comment (caylahamann) [inline dashboards/azure-computer-vision/azure-computer-vision.json:88]: Could this description of the dashboard widget be moved to the top left corner of the dashboard?

File: dashboards/azure-custom-vision-prediction/azure-custom-vision-prediction.json [hunk 4]
```diff
@@ -0,0 +1,323 @@
...
+            "row": 1,
+            "width": 6,
+            "height": 3
+          },
+          "linkedEntityGuids": null,
+          "visualization": {
+            "id": "viz.markdown"
+          },
+          "rawConfiguration": {
+            "text": "# Azure Custom Vision Prediction\nAzure Custom Vision is an image recognition service that lets you build, deploy, and improve your own image identifier models. An image identifier applies labels to images, according to their visual characteristics."
```
Comment (caylahamann) [inline dashboards/azure-custom-vision-prediction/azure-custom-vision-prediction.json:88]: Same thing here, could it be moved to the top left corner?

File: dashboards/azure-custom-vision-training/azure-custom-vision-training.json [hunk 5]
```diff
@@ -0,0 +1,350 @@
...
+            "row": 1,
+            "width": 6,
+            "height": 3
+          },
+          "linkedEntityGuids": null,
+          "visualization": {
+            "id": "viz.markdown"
+          },
+          "rawConfiguration": {
+            "text": "# Azure Custom Vision Training\nAzure Custom Vision is an image recognition service that lets you build, deploy, and improve your own image identifier models. An image identifier applies labels to images, according to their visual characteristics."
```
Comment (caylahamann) [inline dashboards/azure-custom-vision-training/azure-custom-vision-training.json:88]: Same thing here, could it be moved to the top left corner?

File: dashboards/azure-face-api/azure-face-api.json [hunk 6]
```diff
@@ -0,0 +1,389 @@
...
+            "column": 7,
+            "row": 1,
+            "width": 6,
+            "height": 3
+          },
+          "linkedEntityGuids": null,
+          "visualization": {
+            "id": "viz.markdown"
+          },
+          "rawConfiguration": {
```
Comment (caylahamann) [inline dashboards/azure-face-api/azure-face-api.json:87]: also here

**Review by caylahamann — COMMENTED** [review]
Could you move the markdown widgets to the top left hand corner? I commented on most places but there's a lot of dashboards

File: quickstarts/azure/azure-custom-vision-prediction/config.yml [hunk 7]
```diff
@@ -0,0 +1,40 @@
+slug: azure-custom-vision-Prediction
```
Comment (caylahamann) [inline quickstarts/azure/azure-custom-vision-prediction/config.yml:1]:
```suggestion
slug: azure-custom-vision-prediction
```

File: dashboards/azure-anomaly-detector/azure-anomaly-detector.json (hunk 2 above)
Comment (aj-NR) [reply dashboards/azure-anomaly-detector/azure-anomaly-detector.json:88]: this is the standard being followed across all azure entities for project northstar which are being merged and live. We intend to follow similar template for the rest of the services

File: dashboards/azure-content-moderator/azure-content-moderator.json [hunk 8]
```diff
@@ -0,0 +1,290 @@
...
+            "column": 7,
+            "row": 1,
+            "width": 6,
+            "height": 2
+          },
+          "linkedEntityGuids": null,
+          "visualization": {
+            "id": "viz.markdown"
+          },
+          "rawConfiguration": {
```
[the remark this answers is not in the document]
Comment (aj-NR) [reply dashboards/azure-content-moderator/azure-content-moderator.json:87]: same as above

File: dashboards/azure-custom-vision-training/azure-custom-vision-training.json (hunk 5 above)
Comment (aj-NR) [reply dashboards/azure-custom-vision-training/azure-custom-vision-training.json:88]: same as above

File: dashboards/azure-face-api/azure-face-api.json (hunk 6 above)
Comment (aj-NR) [reply dashboards/azure-face-api/azure-face-api.json:87]: same as above

File: quickstarts/azure/azure-custom-vision-prediction/config.yml (hunk 7 above)
Comment (aj-NR) [reply quickstarts/azure/azure-custom-vision-prediction/config.yml]: done

File: dashboards/azure-anomaly-detector/azure-anomaly-detector.json (hunk 1 above)
Comment (aj-NR) [reply dashboards/azure-anomaly-detector/azure-anomaly-detector.json:3]: for azure services, none of the dashboard has description so far, following the same standard

**Review by caylahamann — APPROVED** [review]

**Pull request merged**